src/TestProblem/Hydro/Riemann/Init_TestProb_Hydro_Riemann.cpp

```diff
@@ -525,6 +538,10 @@ void Init_TestProb_Hydro_Riemann()
 #  ifdef MHD
    Init_Function_BField_User_Ptr = SetBFieldIC;
 #  endif
+#  if ( EOS == EOS_NUCLEAR  &&  NUC_TABLE_MODE == NUC_TABLE_MODE_TEMP )
```

Comment:
Why use `EOS == EOS_NUCLEAR  &&  NUC_TABLE_MODE == NUC_TABLE_MODE_TEMP` here, but `TEMP_IG` in `Init_TestProb_Hydro_CCSN.cpp`?